questions/qa-http-and-lang.de.html

```diff
@@ -47,116 +47,122 @@
+  </section>
+
+  <section>
+    <h3 id="internal_meta"><a href="#internal_meta">Dokumentinterne Metadaten</a></h3>
+    <p>Eine Implikation davon, dass HTML den Einsatz des <code class="kw" translate="no">meta</code>-Elements zur Sprachkennzeichnung fallen lässt, ist dass es nun keinen offensichtlichen Weg gibt, Metadaten über ein Dokument innerhalb des Dokuments anzugeben. Theoretisch wäre dies nützlich für Content-Management-Systeme, Übersetzungsprozesse und ähnliches. Diese Art der Information kann über einen HTTP-Header übertragen werden (wie wir im nächsten Abschnitt sehen werden), aber solche Systeme und Prozesse neigen dazu, mit Dokumenten zu arbeiten, die nicht von einem Server unter Angabe eines HTTP-Headers ausgeliefert werden, wofür Metadaten innerhalb des Dokuments nützlich wären.</p>
+    <p>Vielleicht kann ein anderer Ansatz wie <a class="print" href="http://wikipedia.org/wiki/RDFa">RDFa</a> eine Möglichkeit darstellen, solche Informationen zukünftig abzubilden.</p>
+    <p>Die <a rel="noopener" href="https://wiki.whatwg.org/wiki/MetaExtensions" class="external external-icon">MetaExtensions-Seite des WHATWG-Wikis</a> bietet eine erweiterte Liste von Werten, die als <code class="kw" translate="no">name</code>-Wert des <code class="kw" translate="no">meta</code>-Elements verwendet werden können, obwohl keiner davon bisher akzeptiert wurde. Einer dieser Werte ist <code translate="no">dc.language</code>, der dazu dient, Sprache im Rahmen von <a class="print" href="http://en.wikipedia.org/wiki/Dublin_core">Dublin Core</a> auszudrücken.</p>
+    <figure class="example">
+      <div class="badcode"><span><img src="../icons/dontcopy.png" alt=" "> Schlechter Code. Nicht kopieren!</span></div>
+      <p><code>&lt;meta name="dc.language" content="en"&gt;</code></p>
+    </figure>
+    <p>Es ist unklar, ob diese Information je von Browsern beachtet wird, oder in welchem Umfang sie von anderen Anwendungen benutzt würde. Die WHATWG-Seite empfiehlt, statt dessen das <code class="kw" translate="no">lang</code>-Attribut zu verwenden. Diese Empfehlung ist gut, um die Textverarbeitungssprache zu deklarieren, aber gibt nicht ihren möglichen Gebrauch als Ausdruck von Metadaten über eine Seite als Ganzes wieder.</p>
+  </section>
 </section>
-</section>
-
 
 <section>
-<h2 id="endlinks"><a href="#endlinks">Further reading</a></h2>
-<aside class="section" id="survey"> </aside><script>document.getElementById('survey').innerHTML = g.survey</script>
-
-<ul id="full-links">
+  <h2 id="endlinks" class="notoc"><a href="#endlinks">Literaturhinweise</a></h2>
+  <aside class="section" id="survey"></aside>
+  <script>
+    document.getElementById('survey').innerHTML = g.survey
+  </script>
+  <ul id="full-links">
     <li>
-      <p>Erste Schritte: <a href="/International/getting-started/language"><cite>Sprachangaben im Web</cite></a></p>
+      <p>Erste Schritte: <a href="https://www.w3.org/International/getting-started/language"><cite>Sprachangaben im Web
```

Comment:
Although I don't know why, https://www.w3.org/International/i18n-drafts/pages/translation#checklist says "Links to files under https://www.w3.org/International start with /International".

Reply:
Interesting. This is coming straight from the original version (FYI @r12a). I trimmed this and other links as the translation checklist is explicit about this point. Thanks!